Graphics/GraphicsEngineOpenGL/include/VAOCache.hpp

```diff
@@ -85,8 +85,8 @@ class VAOCache
         // PSO uniquely defines the layout (attrib pointers, divisors, etc.),
         // so we do not need to add individual layout elements to the key.
         // The key needs to contain all bound buffers.
-        const UniqueIdentifier PsoUId;
-        const UniqueIdentifier IndexBufferUId;
+        UniqueIdentifier PsoUId;
+        UniqueIdentifier IndexBufferUId;
```

Comment:
Can you add default initialization for these members to make sure there is no garabage?

Reply:
What about `Streams`? There is a comment "Do not zero-out", I assumed the lack of default initialization is intentional.